transport/http/handler.go

```diff
@@ -147,13 +149,11 @@ func handleOnewayRequest(
 	return nil
 }
 
-func updateSpanWithErr(span opentracing.Span, err error) error {
+func updateSpanWithErr(span opentracing.Span, err error) {
```

Comment:
I see. I'm okay with leaving this as-is since it can still be called regardless of `err != nil`.